bin/moonpick

```diff
@@ -20,45 +20,79 @@ Usage: moonpick [opts] <file>, [file2, ..]
 
 Where [opts] can be the following:
 
--v Shows moonpick version
--h Shows this help
+-v                Shows moonpick version
+-h                Shows this help
+--filename file   Filename used for lint_config with stdin
+
+--                Read from stdin, must be last argument
```

Comment:
I've only ever seen stdin being indicated via arguments using a single dash, `-`, so let's use that instead of `--` (which is typically used for indicating the end of passed flags).